config/initializers/after_commit_queue_hacks.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+AfterCommitSubscriber::AfterCommitCallback.class_eval do
+  def before_committed!
+    # noop
+  end
+end
```

Comment:
no idea why but i will just believe you on this one :joy:

Reply:
It is because all callbacks class should have `before_committed!` method otherwise Rails will complain but it is already extracted https://github.com/3scale/porta/commit/eebaa23a3228ec75025076d5d34979937a6a492b